### PR TITLE
Accordion

### LINKED ISSUE #64: [Product] Implement Accordion
Implement accordion for FAQ like:

Screenshot 2023-03-13 at 14 55 02

Example:
* https://www.moleculardevices.com/en/assets/tutorials-videos/dd/cns/enhance-your-electrophysiology-studies-with-axon-pclamp-11-software
* https://main--moleculardevices--hlxsites.hlx.page/applications/monoclonal-antibodies-mabs/hybridoma
* https://main--moleculardevices--hlxsites.hlx.page/technology/electrophysiology

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #64

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/dd/cns/enhance-your-electrophysiology-studies-with-axon-pclamp-11-software
- After: https://accordion--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/dd/cns/enhance-your-electrophysiology-studies-with-axon-pclamp-11-software

AND

- Before: https://main--moleculardevices--hlxsites.hlx.page/technology/electrophysiology
- After: https://accordion--moleculardevices--hlxsites.hlx.page/technology/electrophysiology

AND

- Before: https://main--moleculardevices--hlxsites.hlx.page/applications/monoclonal-antibodies-mabs/hybridoma
- After: https://accordion--moleculardevices--hlxsites.hlx.page/applications/monoclonal-antibodies-mabs/hybridoma

### CODE REVIEW

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -123,6 +123,18 @@ export function toCamelCase(name) {
...
+ * @param {string} name The icon file name (minus ".svg")
+ * @returns {HTMLSpanElement}
+ */
+export function createIcon(name) {
+  const icon = document.createElement('span');
+  icon.classList.add('icon', `icon-${name}`);
+
+  return icon;
+}
+
```
Comment (mhaack) [inline scripts/lib-franklin.js:137]: Is this really needed can be achieved with the dom helpers as well easily

File: blocks/accordion/accordion.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  border: 1px solid var(--background-color-green);
+  border-top-right-radius: 3px;
+}
+
+.blue .accordion-trigger .number {
+  background-color: var(--background-color-blue);
+}
+
+.blue.numbers .accordion-item:not([aria-expanded]) .accordion-trigger,
+.blue.products .accordion-item:not([aria-expanded]) .accordion-trigger {
```
Comment (mhaack) [inline blocks/accordion/accordion.css:53]: Do we really need the `products` & `numbers` styles extra here? Why is `.blue` not enough to set the colors?

File: blocks/accordion/accordion.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  color: var(--background-color-blue);
+}
+
+.green .accordion-trigger .number {
+  background-color: var(--background-color-green)
+}
+
+.green.numbers .accordion-item:not([aria-expanded]) .accordion-trigger,
+.green.products .accordion-item:not([aria-expanded]) .accordion-trigger {
+  color: var(--background-color-green);
```
Comment (mhaack) [inline blocks/accordion/accordion.css:63]: Do we realy need the `products` & `numbers` styles extra here? Why is `.blue` not enough to set the colors?

File: blocks/accordion/accordion.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
+.accordion-trigger {
```
Comment (mhaack) [inline blocks/accordion/accordion.css:1]: Please start all block specific classes always with the block selector, here `.accordion`.

**Comment by mhaack** [comment]
Also compare the product version with the original. That has 2 column with image left and all text right style.

File: scripts/lib-franklin.js (hunk 1 above)
Comment (lydiapuric) [reply scripts/lib-franklin.js]: Removed.

File: blocks/accordion/accordion.css (hunk 3 above)
Comment (lydiapuric) [reply blocks/accordion/accordion.css:48]: products and green themes got removed

File: blocks/accordion/accordion.css [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  display: flex;
+  background-color: #f6f6f6;
```
Comment (mhaack) [inline blocks/accordion/accordion.css:3]: this should maybe be already a color var?

File: blocks/accordion/accordion.css [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  color: #454545;
+  border: 1px solid #c5c5c5;
```
Comment (mhaack) [inline blocks/accordion/accordion.css:5]: this should maybe be already a color var?

File: blocks/accordion/accordion.css [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+[aria-expanded] .accordion-trigger {
+  background-color: var(--background-color-blue);
+  color: var(--text-white);
+  border: 1px solid var(--background-color-blue);
+  border-top-right-radius: 3px;
+}
+
+.accordion-trigger .number {
+  background-color: var(--background-color-green);
+  border: 1px solid #fff;
```
Comment (mhaack) [inline blocks/accordion/accordion.css:29]: this should maybe be already a color var?

**Review by mhaack — APPROVED** [review]

**Review by duynguyen — APPROVED** [review]

**Pull request merged**